build

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+#
+# Build steps to validate client_wrapper.
+
+( # Run unit tests and calculate code coverage.
+  coverage run --source client_wrapper -m unittest discover
+) &&
+( # Check that source has correct formatting.
+  yapf --diff --recursive --style google ./
+) &&
+( # Run static analysis for Python bugs/cruft.
+  pyflakes client_wrapper/*.py tests/*.py
+)
```

Comment:
Since these commands are all running sequentially, is there a reason to have them running in subshells (by wrapping them parens)?  I think you can get away with simply this:

```
coverage run --source client_wrapper -m unittest discover &&
yapf --diff --recursive --style google ./ &&
pyflakes client_wrapper/*.py tests/*.py
```